### DISCUSSION TITLE
v2.0.2-rc

### DISCUSSION THREAD

**Discussion by aaronhelton** [dag-hammarskjold-library/dlx-rest#1031] [General]
## What's Changed
* add search box to browse results by @aaronhelton in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1025 (Addresses issue #858 )
* don't clean auth-controlled values in bib; exception handling by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1027 (Addresses issue #1023 )
* Auth updates and browse indexes by @jbukhari in https://github.com/dag-hammarskjold-library/dlx-rest/pull/1028 (Addresses issues #882 #986 and #1018 )

**Full Changelog**: https://github.com/dag-hammarskjold-library/dlx-rest/compare/v2.0.1...v2.0.2-rc

<hr /><em>This discussion was created from the release <a href='https://github.com/dag-hammarskjold-library/dlx-rest/releases/tag/v2.0.2-rc'>v2.0.2-rc</a>.</em>

**Comment by aaronhelton** [comment]
Deployed to QAT 2023-03-22

**Reply by viola-v** [reply]
Tested everything and as far as I can tell it all works as intended - not 100% sure about #1018 because this particular record isn't in QAT...

**Comment by viola-v** [comment]
Tested everything in UAT and everything still looks good! Also performed regression testing as per the suggestions below:

Regression testing implications for 2.0.2: Delete or merge an authority record and note whether it is deleted from the browse indexes (882); and merge two authorities where the authority type will be both a subject and an author (986) and note that both indexes are updated with the merged variant.
